src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        final int SizeArray = 6;
```

Comment:
This field can be a constant of class

Reply:
I got message: "Modifier 'static' not allowed here".